resources/js/components/MartyrsSlider.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <div class="section" data-appear-animation="fadeInDown" data-appear-animation-delay="1150">
+        <h3 class="section_title">{{ config.title }}</h3>
+        <ul class="writers_list clearfix" ref="owlContainer">
+            <li v-for="martyr in config.martyrs" :key="martyr.name">
+                <a :href="martyr.href">
+                    <div>
+                        <img :src="martyr.img" :alt="martyr.name">
+                    </div>
+                    <div class="post_text">
+                        <h4>{{ martyr.name }}</h4>
+                        <div class="event_date">{{ martyr.city }}</div>
+                    </div>
+                </a>
+            </li>
+        </ul>
+    </div>
+</template>
+
+<style scoped lang="scss">
+ul.writers_list li {
+    margin-left: 0;
+}
+
+</style>
```

Comment:
i prefer this to be at the end of the component and remove that extra empty line